spec.html

```diff
@@ -19727,6 +19992,31 @@ <h1>Static Semantics: Early Errors</h1>
           It is a Syntax Error if PropName of |MethodDefinition| is `"prototype"`.
         </li>
       </ul>
+      <emu-grammar>
+        FieldDefinition :
+          ClassElementName Initializer?
+      </emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if ContainsArguments of |Initializer| is *true*.</li>
+        <li>It is a Syntax Error if |Initializer| is present and |Initializer| Contains |SuperCall| is *true*.</li>
+      </ul>
+
+      <emu-grammar>
+        ClassBody : ClassElementList
+      </emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if PrivateBoundIdentifiers of |ClassBody| contains any duplicate entries.</li>
+      </ul>
+
+      <emu-grammar>ClassElement : FieldDefinition `;`</emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if PropName of |FieldDefinition| is `"constructor"`.</li>
```

Comment:
`class { static constructor() {} }` is allowed; does this imply that `class { static constructor; }` is disallowed? or is it only disallowing `class { constructor; }`?

Reply:
This PR does not introduce static fields. Seems like the proposal that does introduce disallows `class { static constructor; }`, though. I'd suggest filing an issue on https://github.com/tc39/proposal-static-class-features.

Reply:
@Ms2ger "class fields" has been a single proposal since before stage 3; all of it should be added in a single PR - static and instance, public and private.

Reply:
I don't understand what makes you say that. There's three specifications, and there's three proposals listed in https://tc39.es/#proposals.